typings/commander-tests.ts

```diff
@@ -81,6 +81,7 @@ const optionThis1: commander.Command = program.option('-a,--alpha');
 const optionThis2: commander.Command = program.option('-p, --peppers', 'Add peppers');
 const optionThis3: commander.Command = program.option('-s, --string [value]', 'default string', 'value');
 const optionThis4: commander.Command = program.option('-b, --boolean', 'default boolean', false);
+const optionThis5: commander.Command = program.option('-o, --object [value]', 'default object', {key: 'value'});
```

Comment:
I think you should be testing with a custom coercion function, which would be the normal usage case.

This could go after the "readme" coercion tests in a new group, and avoid the change churn in the variable numbering.  🙂